misc/config_tools/scenario_config/pci_dev_c.py

```diff
@@ -131,12 +131,12 @@ def generate_file(vm_info, config):
 
     print("{}".format(scenario_cfg_lib.HEADER_LICENSE), file=config)
     print("", file=config)
-    print("#include <x86/vm_config.h>", file=config)
+    print("#include <asm/vm_config.h>", file=config)
     print("#include <pci_devices.h>", file=config)
     print("#include <vpci.h>", file=config)
     print("#include <vbar_base.h>", file=config)
-    print("#include <x86/mmu.h>", file=config)
-    print("#include <x86/page.h>", file=config)
+    print("#include <asm/mmu.h>", file=config)
+    print("#include <asm/page.h>", file=config)
```

Comment:
Do you regenerate the files under  misc\config_tools\data\generic_board\generic_code\<scenario_name> to reflect your changes of config_tools?